### DISCUSSION TITLE
How to update serviceMonitorSelectorNilUsesHelm values when using OpenShift Prometheus Operators is used

### DISCUSSION THREAD

**Discussion by ajay15283** [aquasecurity/trivy-operator#1329] [Help & Support]
If prometheus is not installed from kube-prometheus-stack and instead openshift promethus operators were used in that case how to update serviceMonitorSelectorNilUsesHelmValues to false as the metrics are not visible in grafana dashboard 

[
](https://aquasecurity.github.io/trivy-operator/v0.11.0/tutorials/grafana-dashboard/)

**From Documentation** 
Our Prometheus installation needs to be slightly customised to discover ServiceMonitors by default. Create a values.yaml file with the following configuration:

prometheus:
  prometheusSpec:
    serviceMonitorSelectorNilUsesHelmValues: true
    serviceMonitorSelector: {}
    serviceMonitorNamespaceSelector: {}

**Comment by chen-keinan** [comment]
@ajay15283 I'm converting this discussion to issue #1330 .
